### PR TITLE
ENH: add a first draft for the JOSS submission

### PR DESCRIPTION
**Pull request by ev-br**
As discussed during the dev hangout on 14 Oct (https://docs.google.com/document/d/1srVFUvxEUOKzJBuCI8oqn17D7TYLcLIwMJKFRRD4bB0/edit?ts=57ff506a), and several off-list discussions, here's a first draft for the "abstract-paper" for submission to JOSS, http://joss.theoj.org/

Comments welcome.

Suggested procedure:

* Limit the scope by the 0.18.x series, i.e. releases 0.18.0 and 0.18.1
* Anyone who made a substantial contribution to either of these releases is invited to add themselves as an author. Here "substantial" is interpreted as anything beyond a one-line doc fix.
* Authors can either send PRs adding their names and affiliations to `paper.md`, or add this information to a special issue (to avoid a rebase mess where everyone is editing the same three lines): gh-5
* We need to pick a submission date. Here's one random date, 12 Dec 2016 (three weeks and a weekend from today).

### CODE REVIEW

File: joss_paper_0.18/paper.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+for numerical integration and optimization. 
+
+The latest stable release of SciPy is [0.18.1](@SciPy0181), which improves on the feature
+release [0.18.0](SciPy0180). Together they contain numerous bug-fixes, improved test coverage,
+better documentation and many new features. For the complete description of the
+contents of the 0.18 series, see the [release notes](https://github.com/scipy/scipy/releases/tag/v0.18.0).
+
+# Acknowledgements
+
+We gratefull acknowledge contributions of more than four hundred people whose work
```
Comment (person142) [inline joss_paper_0.18/paper.md]: Typo gratefull -> gratefully

File: joss_paper_0.18/paper.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+and more. 
+
+The @SciPy library depends on @NumPy, which provides convenient and fast N-dimensional
+array manipulation. The SciPy library is built to work with NumPy arrays, and
+provides many user-friendly and efficient numerical routines such as routines
+for numerical integration and optimization. 
+
+The latest stable release of SciPy is [0.18.1](@SciPy0181), which improves on the feature
+release [0.18.0](SciPy0180). Together they contain numerous bug-fixes, improved test coverage,
+better documentation and many new features. For the complete description of the
```
Comment (person142) [inline joss_paper_0.18/paper.md]: documentation and many -> documentation, and many

to be consistent with the other Oxford commas

**Comment by person142** [comment]
Looks good. Should it mention somewhere that the user interface is in Python?

Left small comments which, now that I think of it, might be premature optimizations. (Sorry...)

**Comment by ev-br** [comment]
Premature optimizations or not, I've edited them in. Thanks Josh --- I owe you a beverage of your choice next time we're in the same city.

File: joss_paper_0.18/paper.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ - name: Andrew Nelson
+   orcid: 0000-0002-4548-3558
+   affiliation: Australian Nuclear Science and Technology Organisation
+date: XXX
+bibliography: paper.bib
+---
+
+# Summary
+
+SciPy (pronounced "Sigh Pie") is open-source library for mathematics, science,
```
Comment (rgommers) [inline joss_paper_0.18/paper.md]: ``open-source`` --> ``an open source``

File: joss_paper_0.18/paper.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+The @SciPy library depends on @NumPy, which provides convenient and fast N-dimensional
+array manipulation. The SciPy library is built to work with NumPy arrays, and
+provides many user-friendly and efficient numerical methods such as routines
+for numerical integration and optimization. 
+
+The latest stable release of SciPy is [0.18.1](@SciPy0181), which improves on the feature
+release [0.18.0](SciPy0180). Together they contain numerous bug-fixes, improved test coverage,
+better documentation, and many new features. For the complete description of the
+contents of the 0.18 series, see the [release notes](https://github.com/scipy/scipy/releases/tag/v0.18.0).
```
Comment (rgommers) [inline joss_paper_0.18/paper.md]: Did you consider adding a selection of highlights? Or is even that too much? The "bug fixes, improved test coverage, ..." feels slightly too generic.

From the release notes:

Highlights of this release include:

- A new ODE solver for two-point boundary value problems,
  `scipy.optimize.solve_bvp`.
- A new class, `CubicSpline`, for cubic spline interpolation of data.
- N-dimensional tensor product polynomials, `scipy.interpolate.NdPPoly`.
- Spherical Voronoi diagrams, `scipy.spatial.SphericalVoronoi`.
- Support for discrete-time linear systems, `scipy.signal.dlti`.

File: joss_paper_0.18/paper.bib [hunk 5]
```diff
@@ -0,0 +1,38 @@
+@SciPy{,
+  author =    {Eric Jones and Travis Oliphant and Pearu Peterson and others},
```
Comment (rgommers) [inline joss_paper_0.18/paper.bib]: This is taken from http://scipy.org/citing.html. Should we keep it like that, or update according to the author ranking of Zenodo for example? This is not an actual fixed citation after all.

File: joss_paper_0.18/paper.bib [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  title =     {{SciPy}: Open source scientific tools for {Python}},
+  year =      {2001--},
+  url = "http://www.scipy.org/"
+}
+
+@Article{NumPy,
+  author =       {Stéfan {van der Walt} and S. Chris Colbert and
+                  Gaël Varoquaux},
+  title =        {The NumPy Array: A Structure for Efficient Numerical
+                  Computation},
```
Comment (rgommers) [inline joss_paper_0.18/paper.bib]: Also from http://scipy.org/citing.html. I'd think that instead we should cite the library itself, in similar style to the SciPy citation above.

**Comment by rgommers** [comment]
Thanks for getting the ball rolling on this Evgeni!

File: joss_paper_0.18/paper.bib (hunk 6 above)
Comment (ev-br) [inline joss_paper_0.18/paper.bib]: Precisely, I followed the recommendation in  http://scipy.org/citing.html. So you're suggesting to change the recommendation over there as well? If that's changed, I guess the right thing to do would be to cite both the library itself and this paper.

**Comment by ev-br** [comment]
Updated with an expanded list of subpackages and highlights from the release notes, per Ralf's suggestions.

File: joss_paper_0.18/paper.bib (hunk 5 above)
Comment (rgommers) [inline joss_paper_0.18/paper.bib]: That's because Google Scholar keeps track of any changes, there's no such citable paper or item with a DOI anywhere

File: joss_paper_0.18/paper.bib (hunk 6 above)
Comment (rgommers) [inline joss_paper_0.18/paper.bib]: Actually, I'm much more convinced of changing the NumPy one rather than the SciPy one. A citation for NumPy that doesn't include Travis and Chuck makes little sense.

**Comment by ev-br** [comment]
Updated to cite numpy two ways: both the 2011 paper and the library itself, https://github.com/scipy/scipy-articles/pull/4#discussion_r88737331

File: joss_paper_0.18/paper.bib (hunk 6 above)
Comment (charris) [inline joss_paper_0.18/paper.bib]: I'd certainly put Travis first in the list of authors.

**Comment by rgommers** [comment]
Closing this, we are not going to finish a 0.18.x paper. We could reuse some of the content in this paper for a future JOSS paper possibly, but no need to keep the PR open for that.
